.github/CODE_OF_CONDUCT.md

```diff
@@ -8,5 +8,5 @@ accepted and unaccepted behavior.
 Please read also the [reporting guidelines][guidelines], in case you encountered
```

Comment:
I think we can remove this file, in favor of the one in https://github.com/symfony/.github ?

Reply:
You're right. 👍 
I removed it.